### ISSUE TITLE
Do you know the loss of siamrpn++ in detail?

### ISSUE THREAD

**Issue by ghost** [JohnPekl/siamrpnpp#1]
Do you know the loss of siamrpn++ in detail?
可以给出损失函数的具体公式吗？

**Comment by iamlvm** [comment]
Hi, the loss function of siamrpn++ is the same as the loss function of siamprn.
You can refer siamrpn loss function in the following [github](https://github.com/makalo/Siamese-RPN-tensorflow).
